mods/ws_core/tools.lua

```diff
@@ -21,10 +21,12 @@ minetest.register_tool("ws_core:knife_flint", {
 		full_punch_interval = 1.2,
 		max_drop_level=0,
 		groupcaps={
-			cracky = {times={[2]=4.0, [3]=1.25}, uses=15, maxlevel=1},
+			snappy = {times={[1]=0.40}, uses=50, maxlevel=0},
+			choppy = {times={[1]=3.00, [2]=4.00}, uses=30, maxlevel=0},

```

Comment:
Breaks compatibility with other mods, never do that. I think it's possible to write transformer that automatically transforms `ws_snappy = 1` to `snappy = 3`, but I'm not sure if that's a good solution

Reply:
Can I sway your opinion on this by mentioning that almost every mod this would break, will be already broken by the fact that we don't use default? Most mods will need _some_ degree of adjustment so they can work with this game. So it will be more the _degree_ of incompatibility, rather then compatibility being broken at all.

I feel like MTG is stagnating prety hard, because every proposal for significant change gets shut down, to keep compability with all the mods written for it.
Since WS is still in the alpha phase and only very little depends on its current implementation, now is the time when we _can_ make significant changes to the ground mechanics, if we are going to make them at all.

My reasons for wanting these changes I already elaborated.

PS: Automatic group redefinitions sound like a bad plan indeed.

Reply:
Fixing `default` is pretty easy (see: [this is how I did that for Trinium](https://gitlab.com/MultiDragon/trinium/-/tree/master/mods/default/code) - this fixes 99% of mods). Fixing this is very hard because it basically needs automatic redefinitions. And the payoff for that is insufficient imo.

Reply:
You basicaly made an interface so other mods can work with your game as if it _is_ default?
I don't realy use mods much, so I suppose I just didn't think about how much of a difference it would make.